src/commands/vote-mockup.ts

```diff
@@ -19,8 +19,24 @@ export default class VoteMockupCommand extends SlashCommand<Client> {
 
     const game = games.get(ctx.channelID);
 
+    const start = Date.now();
+    const duration = 1000 * 20;
+    const ballot = new Map<string, string>();
```

Comment:
As an incentive for state management within the scope, this works... for now. Although options may be explored for using a class structure to manage the data that flows through and the logic of the sequence itself.

```ts
class BallotInstance {
  startedAt: number = Date.now();
  constructor(public game: IGame, public durationMs: number) {

  }

  get remainingTime(): number {
    return this.startedAt + duration - Date.now();
  }

  // etc. etc.
}
```

Reply:
`duration` can also have the possibility of being scaled by the remaining players, but that would likely come with its own problems in how the time scales in a declining trend as the game progresses. [Graph example](https://www.desmos.com/calculator/4alamggh46)
```ts
const gradient = 2.2; // { g }
const offset = 8.1; // { o }

class BallotInstance {
  get remainingPlayerCount(): number { return 8; } // ... as { r }
  get totalPlayerCount(): number { return 15; } // as { x }

  get duration(): number {
    return Math.round(
      (this.totalPlayerCount / this.remainingPlayerCount) * gradiant + offset
    )
  }
}
```
Over time, this would prove to be effective... if the time available for each day were gradually reduced with each cycle. However, the inconsistency of how it functions may confuse players - and in return likely urge them to rush their resulting decision.